### DISCUSSION TITLE
ESP32-8048S070 display flickering when any kind of animation is running.

### DISCUSSION THREAD

**Discussion by Radek-Suski** [moononournation/Arduino_GFX#299] [General]
Hi good people

Firstly; thank you for doing a great job. I really appreciate it.

I have a "small" issue though; I am trying for days to get this library working with lvgl, and I saw [the demo ](https://www.youtube.com/watch?v=7BRGVsnQpgE) working perfectly fine, but for some reason the display is flickering the entire time as soon as something is being animated. 

https://github.com/moononournation/Arduino_GFX/assets/1110663/f727c8ba-4dd0-4d74-a471-997c6b4e2ba6

I am pretty sure I screwed something but I am simply not able to figure out what am I doing wrong.

You can take a look at [my code ](https://code.sigsiu.net/Radek-Suski/PlantMonitor/-/tree/master), most of the relevant stuff is in [src/Libraries/Display.h](https://code.sigsiu.net/Radek-Suski/PlantMonitor/-/blob/master/src/Libraries/Display.cpp)

A side note: I have 3 of these displays and tested the code on all three so I don't think it's a hardware issue.

Thanks in advance for any advice.

Regards,
Radek

**Comment by moononournation** [comment]
RGB screen flickering still a known issue, may need to wait the new enhancement come from IDF 5.

**Reply by Radek-Suski** [reply]
but I am using 5.0.1 as far I can say
